raiden/transfer/mediated_transfer/initiator.py

```diff
@@ -64,9 +64,21 @@ def calculate_safe_amount_with_fee(
     This total amount consists of the payment amount, the estimated fees as well as a
     small margin that is added to increase the likelihood of payments succeeding in
     conditions where channels are used for multiple payments.
+
+    We could get much better margins by considering that we only need margins
+    for imbalance fees. See
+    https://github.com/raiden-network/raiden-services/issues/569.
     """
-    # `max` is taken as `estimated_fee` can be negative
-    fee_margin = round(max(estimated_fee, FeeAmount(0)) * DEFAULT_MEDIATION_FEE_MARGIN)
+    if estimated_fee == 0:
+        # If the total fees are zero, we assume that no fees are set. If the
+        # fees sum up to zero incidentally, we're should add a margin, but we
+        # can't detect that case.
+        return PaymentWithFeeAmount(payment_amount)
+
+    fee_margin = round(
+        abs(estimated_fee) * DEFAULT_MEDIATION_FEE_MARGIN
+        + payment_amount * MAX_MEDIATION_FEE_PERC * 0.01
```

Comment:
Can we make this 1% a constant/setting as well.

Reply:
Naming suggestions welcome! This is a hard one.

Reply:
😅